### PR TITLE
New Package: apbs

### PR DESCRIPTION
**Pull request by qwertos**
Depends on: #16631

### CODE REVIEW

File: var/spack/repos/builtin/packages/apbs/package.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    for large biomolecular assemblages."""
+
+    homepage = "http://www.poissonboltzmann.org/"
+    url      = "https://downloads.sourceforge.net/project/apbs/apbs/apbs-1.3.0/apbs-1.3-source.tar.gz"
+
+    # Working around race condition
+    parallel = False
+
+    # Version 1.4.2.1 uses cmake. Commented out pending resolution of
+    # https://github.com/spack/spack/pull/12941
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/apbs/package.py:20]: Would it be better to comment out 1.3 and keep 1.4+, or do you need 1.4+?

File: var/spack/repos/builtin/packages/apbs/package.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    # version('1.4.2.1', sha256='aa3de78394ccfdbc29115f07fb005480603a4884628b8f82c9e38d93889dc2be',
+    #         url="https://downloads.sourceforge.net/project/apbs/apbs/apbs-1.4.2/apbs-1.4.2.1-source.tar.gz" )
+    version('1.3', sha256='5fa5b597f7d5a3d9bb55429ec4fefc69e7d0f918d568c3c4a288088c0fde9ef2')
+
+    variant('python', default=False, description='Build with python wrappers')
+
+    depends_on('python@:2.9999', when='+python')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/apbs/package.py:27]: Does it actually link to Python? Python is usually build/run type.

File: var/spack/repos/builtin/packages/apbs/package.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        else:
+            args.append('--disable-python')
+
+        return args
+
+    def install(self, spec, prefix):
+        super(Apbs, self).install(spec, prefix)
+        mkdir(prefix.src)
+        cp = which('cp')
+        cp('-r', '{0}/.'.format(self.stage.source_path), prefix.src)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/apbs/package.py:47]:
```suggestion
    @run_after('install')
    def install_src(self):
        install_tree(self.stage.source_path, self.prefix.src)
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/apbs/package.py (hunk 1 above)
Comment (qwertos) [reply var/spack/repos/builtin/packages/apbs/package.py:20]: I keep going back and forth on this matter. Re-looking at the documentation for the [dependent package](https://rashmikumari.github.io/g_mmpbsa/Download-and-Installation.html) I could go to 1.4. I'm going to mark this as draft for now.

**Comment by adamjstewart** [comment]
Is this still a WIP or can it be closed?

**Comment by qwertos** [comment]
I'm gonna abandon this PR. Forgot what I needed apbs for at this point. I may resurrect it at some point.

**Pull request closed without merging**